src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.scss

```diff
@@ -33,3 +33,7 @@
     width: 100%;
   }
 }
+
+::ng-deep .tooltip.show {
+  opacity: 1 !important;
+}
```

Comment:
If we assume that tooltips should be opaque (since transparence is not good for readibility), why doing this at a component level and not in the global stylesheet? We should aim at styling things at the highest level possible. The per-component styling, IMHO is one of most misused Angular features.

Reply:
My thought here was that perhaps it could have been desirable elsewhere so only update the component in scope